packages/peer-exchange/src/waku_peer_exchange_discovery.ts

```diff
@@ -165,6 +219,13 @@ export class PeerExchangeDiscovery
       }
     );
   }
+
+  private backOffPeer(peerIdStr: string, interval: NodeJS.Timer): void {
+    log(`Backing off peer ${peerIdStr}`);
+    clearInterval(interval);
+    this.intervals.delete(peerIdStr);
+    this.queryAttempts.delete(peerIdStr);
+  }
```

Comment:
_backing off_  describes an algorithm that:
- attempts an action until it fails
- the interval between each attempt increases over time

This function does not do that, it aborts the retries.